chsdi/models/vector/are.py

```diff
@@ -39,10 +37,9 @@ class Alpenkonvention(Base, Vector):
     __bodId__ = 'ch.are.alpenkonvention'
     __label__ = 'stand'
     id = Column('row_id', Integer, primary_key=True)
-    the_geom = Column(Geometry(geometry_type='GEOMETRY',
-                               dimension=3, srid=21781))
     flaeche_ha = Column('flaeche_ha', Numeric)
     stand = Column('stand', Numeric)
+    the_geom = Column(Geometry2D)
```

Comment:
I owed you one :)

On 12.05.2016 11:23, Gasser Loïc wrote:
> In chsdi/models/vector/are.py [1]:
> 
>> flaeche_ha = Column('flaeche_ha', Numeric)
>> stand = Column('stand', Numeric)
>> + the_geom = Column(Geometry2D)
> 
> yes good you review
> 
> --
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub [2]
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/geoadmin/mf-chsdi3/pull/2068#discussion_r62989259
> [2]
> https://github.com/geoadmin/mf-chsdi3/pull/2068/files/85fd6e0954c5cfbf411f90da9d783eaddae52356#r62989259